modules/apps/josh-lapurd-theme/src/WEB-INF/src/resources-importer/sitemap.json

```diff
@@ -0,0 +1,120 @@
+{
+	"layoutTemplateId": "1_2_1_columns",
+	"publicPages": [
+		{
+			"columns": [
+				[
+					{
+						"portletId": "com_liferay_journal_content_web_portlet_JournalContentPortlet",
+						"portletPreferences" : {
+							"articleId": "img_Hero.xml",
```

Comment:
Rename article id's to all lower case since we have the underscore to separate the words.

Ex.
```
img_hero.xml
```